addons/spreadsheet/static/tests/pivots/model/pivot_plugin_test.js

```diff
@@ -995,41 +992,6 @@ QUnit.module("spreadsheet > pivot plugin", {}, () => {
         assert.strictEqual(getEvaluatedCell(model, "B1").format, "dd/mm/yyyy");
     });
 
-    QUnit.test("Inserted pivot is inserted with a table", async function (assert) {
-        const { model } = await createSpreadsheetWithPivot();
-        const [pivotId] = model.getters.getPivotIds();
-        const sheetId = model.getters.getActiveSheetId();
-        const pivotZone = getZoneOfInsertedDataSource(model, "pivot", pivotId);
-        const tables = model.getters.getTables(sheetId);
-
-        assert.equal(tables.length, 1);
-        assert.deepEqual(tables[0].range.zone, pivotZone);
-        assert.deepEqual(tables[0].config, { ...PIVOT_TABLE_CONFIG, numberOfHeaders: 1 });
-    });
-
-    QUnit.test(
```

Comment:
RE_INSERT_ODOO_PIVOT should be tested instead, right ?

Reply:
This test is to test the pivot_init_callback. The action to reinsert a pivot is already tested